### ISSUE TITLE
undefined function `conform_escript:main/1` error

### ISSUE THREAD

**Issue by aaronrenner** [bitwalker/conform#125]
I get the following error when trying to start my app after upgrading to 2.3.5

```
_trustedform-queue@app1:~/queue$ ./bin/queue start
escript: exception error: undefined function conform_escript:main/1
  in function  escript:run/2 (escript.erl, line 760)
  in call from escript:start/1 (escript.erl, line 277)
  in call from init:start_em/1
  in call from init:do_boot/3
```

I am running on ubuntu 14.04 and my application was built with edeliver+ distillery. I did not see the issue in 2.3.3. Please let me know what other info would be helpful.

**Comment by bitwalker** [comment]
Oh damn, I know what happened, I built the escript on my machine which has OTP 20 installed, looks like something changed, I'll fix that straight away.

**Comment by bitwalker** [comment]
I've published 2.3.6 and retired 2.3.5, should be good to go now!

**Comment by aaronrenner** [comment]
That fixed it! Thanks! 🎉
